wahoomc/main.py

```diff
@@ -5,6 +5,7 @@
 
 # import official python packages
 import logging
+import time
```

Comment:
nice that we don't log here anymore and don't need the `time` module because this is in the timings class now 👍 
```suggestion
```